core/src/validator.rs

```diff
@@ -1012,6 +1013,28 @@ impl Validator {
             )),
         };
 
+        let vote_connection_cache = match vote_use_quic {
+            true => {
+                let vote_connection_cache = ConnectionCache::new_with_client_options(
+                    "connection_cache_vote_quic",
+                    1,
+                    None,
```

Comment:
Can you please add trailing comments for literal values and what function argument they are, like:
```rust
  1,    // connection_pool_size
  None, // client_endpoint
```
That would make things easier for someone reading the code, and also makes things less error-prone.

Reply:
Done.